### PR TITLE
#444 Test class for XmlMethodArgs

### PR DESCRIPTION
**Pull request by HDouss**
Resolving issue #444 
Added a test class with two cases, and made the class public.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#455](https://github.com/yegor256/jpeek/issues/455) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/0e1c7465-3eec-4862-bd41-0ccc15ebc4a0); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by HDouss** [comment]
@Vatavuk ping

File: src/test/java/org/jpeek/graph/XmlMethodArgsTest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import org.llorllale.cactoos.matchers.Assertion;
+
+/**
+ * Test case for {@link XmlMethodArgs}.
+ * @since 0.30.9
+ */
+public final class XmlMethodArgsTest {
+
+    @Test
+    public void givesArgsForNoArgs() throws IOException {
```
Comment (Vatavuk) [inline src/test/java/org/jpeek/graph/XmlMethodArgsTest.java:41]: @HDouss let's rename the test method to `returnsEmptyStringWhenNoArgsSpecificied` to be more explicite

File: src/test/java/org/jpeek/graph/XmlMethodArgsTest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        final XML method = new Skeleton(new FakeBase("MethodMethodCalls")).xml().nodes(
+            "//method[@name='methodOne']"
+        ).get(0);
+        new Assertion<>(
+            "Must returns args when method has no arguments",
```
Comment (Vatavuk) [inline src/test/java/org/jpeek/graph/XmlMethodArgsTest.java:46]: @HDouss must return empty string when...

File: src/test/java/org/jpeek/graph/XmlMethodArgsTest.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        ).affirm();
+    }
+
+    @Test
+    public void givesArgsForMultipleArgs() throws IOException {
+        final XML method = new Skeleton(new FakeBase("MethodsWithDiffParamTypes")).xml().nodes(
+            "//method[@name='methodThree']"
+        ).get(0);
+        new Assertion<>(
+            "Must returns args when method has multiple arguments",
```
Comment (Vatavuk) [inline src/test/java/org/jpeek/graph/XmlMethodArgsTest.java:58]: @HDouss must serialize args when multiple arguments are specified in the xml file

**Comment by HDouss** [comment]
@Vatavuk Thanks. Fixed!

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by HDouss** [comment]
@paulodamaso How can a test succeds in travis but not in rultor. I have no differences with master that could induce a test failure..

**Comment by HDouss** [comment]
@paulodamaso The problem comes from this PR https://github.com/yegor256/jpeek/pull/451
Why does this PR modified XmlMethodArgs.java, and why did it modify the implementation... :(

**Comment by HDouss** [comment]
@paulodamaso should be good now, please try to merge again.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 23 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Pull request merged**